test/Microsoft.AspNet.Mvc.TagHelpers.Test/ValidationMessageTagHelperTest.cs

```diff
@@ -89,19 +111,24 @@ public class ValidationMessageTagHelperTest
             validationMessageTagHelper.ViewContext = expectedViewContext;
 
             // Act & Assert
-            await validationMessageTagHelper.ProcessAsync(context: null, output: output);
+            await validationMessageTagHelper.ProcessAsync(context, output: output);
```

Comment:
side note: change from `context: null` illustrates one of the downsides in this new approach.  tag helper authors can't ignore the `context` argument value and must separately deal with content from children (`context.GetChildContentAsync()`) and previous tag helpers targeting the same element (`output.PreContent`, `output.Content`, `output.PostContent`).

another downside is related: to check other content, must deal w/ `async`